app/client/cli/actor.go

```diff
@@ -0,0 +1,234 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/big"
+	"regexp"
+	"strings"
+
+	sharedTypes "github.com/pokt-network/pocket/shared/types"
+	"github.com/pokt-network/pocket/utility/types"
+	utilityTypes "github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	subcmdFlags := []cmdOption{func(c *cobra.Command) {
+		c.Flags().StringVar(&pwd, "pwd", "", "passphrase used by the cmd, non empty usage bypass interactive prompt")
+	}}
+	rootCmd.AddCommand(NewActorCommands(subcmdFlags)...)
+}
+
+var (
+	// DISCUSS(team): this should probably come from somewhere else
+	stakingRecommendationAmount = big.NewInt(15100000000)
+	pwd                         string
+)
+
+type (
+	cmdOption   func(*cobra.Command)
+	actorCmdDef struct {
+		Name      string
+		ActorType types.ActorType
+		Options   []cmdOption
+	}
+)
+
+func NewActorCommands(cmdOptions []cmdOption) (cmds []*cobra.Command) {
```

Comment:
I just want to make sure I understand. Since we are only reading the values of these structs, it sounds correct to me to pass struct values and not pointers. 🤔 

Also `cmdOption` is a func, I believe you are talking about `actorCmdDef` (https://github.com/pokt-network/pocket/pull/177/files#diff-d716107ed460c3fd94868b09151dcf893fb08e86b22788c8680546f28c9cd8e0R62) and similar cases

Reply:
Yea, I guess I was micro-optimizing for memory... Let's keep it as is